packages/builder/src/pages/builder/portal/apps/[appId]/index.svelte

```diff
@@ -14,6 +17,18 @@
     }
     return `/${app.devId}`
   }
+
+  // Normally fetched in builder/src/pages/builder/app/[application]/_layout.svelte
+  const fetchScreens = async appId => {
+    if (!appId) return
+
+    store.actions.reset()
+    const pkg = await API.fetchAppPackage(appId)
+    await store.actions.initialise(pkg)
```

Comment:
Rather than initialise the store from the portal here, which causes a pretty large cascade of requests and hydration logic, we can just check the length of `pkg.screens` directly. Much simpler and achieves the same thing.